modules/drivers/velodyne/velodyne_pointcloud/include/velodyne_pointcloud/velodyne_parser.h

```diff
@@ -211,12 +211,8 @@ class VelodyneParser {
   // order point cloud fod IDL by velodyne model
   virtual void order(VPointCloud::Ptr &cloud) = 0;
 
-  const Calibration &get_calibration() {
-    return calibration_;
-  }
-  const double get_last_timestamp() {
-    return last_time_stamp_;
-  }
+  const Calibration &get_calibration() { return calibration_; }
+  const double get_last_timestamp() { return last_time_stamp_; }
```

Comment:
I think const double as return will have a compiling warning. Use double for return type.